slices/base-files.yaml

```diff
@@ -16,14 +16,14 @@ slices:
 
   bin:
     contents:
-      /bin/:
-      /sbin/:
```

Comment:
Out of curiosity, why are we removing the trailing slash here? It seems like we are using it below, e.g. `/usr/bin/:`

Reply:
these are symlinks now

Reply:
That means that it is extracting symlinks included in the tarball?

Reply:
Yes, exactly

Reply:
Please re-open if you think this should be done differently :)